docs/en/sdk-manual/migrate-from-third-party-sdks.md

```diff
@@ -0,0 +1,122 @@
+# Migrate from third-party SDKs
+
+## Introduction
+
+Generally, in order not to affect your normal business analysis, when migrating from a third-party SDK to Clickstream SDK, we recommend that you integrate your existing SDK with Clickstream SDK for a period of time, and then remove it when you think that Clickstream SDK can replace the previous SDK.
+
+This article mainly introduces how to integrate Clickstream SDK and third-party SDK into your application at the same time. After understanding this article, you will be able to synchronously integrate third-party SDKs with the Clickstream SDK with minimal code modifications, which will help you transition your data analytics business from other SDKs to Clickstream more smoothly.
+
+To make it easier to understand, we will use Clickstream Web SDK as an example to introduce, for third-party analytics SDK, we take Firebase SDK as an example to introduce, and assume that you have integrated Firebase Web SDK.
```

Comment:
To make it easier to understand, we will use Clickstream Web SDK to replace Firebase Web SDK (GA4 SDK) as an example to illustrate. Assuming that you have integrated Firebase Web SDK in your website, you can follow below steps.

Reply:
changed to this